.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-node-template-fuzzers:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup dependencies
+        run: |
+          cargo install ziggy cargo-afl honggfuzz grcov
+          rustup target add wasm32-unknown-unknown
+      - name: Build fuzzer
+        run: |
+          cargo afl config --build --force
```

Comment:
This won't work in GitHub Actions. You can use these flags instead:

https://github.com/srlabs/ziggy/blob/0496f1e5adfd29d912c50a1943650f3c11f44da6/tests/url_fuzz.rs#L67-L68